packages/tc-ui/src/lib/mappers/component-assigner.js

```diff
@@ -0,0 +1,45 @@
+const {
```

Comment:
Slightly prefer to export `{ componentAssigner }` not `componentAssigner`.

Dora persuaded me that this means there is greater consistency when importing, easier refactoring if you need to add more exports to the same file, and can be easier to write tests for